Android.mk

```diff
@@ -1,5 +1,7 @@
 LOCAL_PATH:= $(call my-dir)
 
+USE_32BIT_MINIAF := $(shell $(LOCAL_PATH)/use_32bit_miniaf.sh $(LOCAL_PATH))
```

Comment:
I think it should be enough to have droid-make with audioflingerglue_32 ? This script functionality could be then moved to .spec file similarly as the LIB_TARGET magic, instead of using shell scripts etc.

Reply:
unfortunately that is not possible as far as i know. if you want to compile a binary (miniafservice in this case) as 32 bits on a 64 bit device you have to force it in Android.mk

but i got rid of the script